backend/src/packages/projects/project.service.ts

```diff
@@ -31,13 +33,14 @@ class ProjectService implements Omit<IService, 'find' | 'update' | 'delete'> {
     };
   }
 
-  public async create(
+  public async createByUserId(
```

Comment:
```suggestion
  public async create(
```
it just a create method